src/lms-ds-loader/lms_ds_loader/arguments.py

```diff
@@ -0,0 +1,47 @@
+# SPDX-License-Identifier: Apache-2.0
+# Licensed to the Ed-Fi Alliance under one or more agreements.
+# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+# See the LICENSE and NOTICES files in the project root for more information.
+
+
+class DbConnection:
+    @staticmethod
+    def build_for_mssql_with_integrated_security(server, port, db_name):
+        assert server is not None
+        assert server.strip() != ""
+
+        assert db_name is not None
+        assert db_name.strip() != ""
+
+        if not port:
+            port = 1433
+        if type(port) == str and port.strip() == "":
+            port = 1433
+
+        return f"mssql+pyodbc://{server},{port}/{db_name}?driver=SQL Server?Trusted_Connection=yes"
+
+    @staticmethod
+    def build_for_mssql(server, port, db_name, username, password):
+        assert server is not None
+        assert server.strip() != ""
+
+        assert db_name is not None
+        assert db_name.strip() != ""
+
+        if not port:
+            port = 1433
+        if type(port) == str and port.strip() == "":
+            port = 1433
+
+        return f"mssql+pyodbc://{username}:{password}@{server},{port}/{db_name}?driver=SQL Server"
+
+
+class Arguments:
+    csv_path = ""
+    engine = ""
+    db_connection = None

```

Comment:
Are you aware that this variables don't referenciate the instance ones? They will act similar to "static" values from the class, while the ones defined in the __init__ method, will act as class instance's attributes.

Reply:
Hm, that was not the impression I had, but I probably wasn't reading carefully. I definitely want instance methods here. Will look into this further.

Reply:
I removed these, only setting them in the constructor